components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Testing/src/Servers/Common/MonitoredNode.cs

```diff
@@ -27,49 +27,58 @@
  * http://opcfoundation.org/License/MIT/1.00/
  * ======================================================================*/
 
-namespace Opc.Ua.Sample {
-    using System.Collections.Generic;
-    using Opc.Ua.Server;
+using System;
```

Comment:
`using` statements should come after `namespace` definition.

Reply:
The thing here is that the test server code is copied 1:1 from the stack's code. If we start applying our coding guidelines on this code it will make future updates really hard to compare.

Reply:
will do this outside of this PR